localstack/services/events/provider.py

```diff
@@ -698,3 +712,35 @@ def _process_entries(
                             )
                             failed_entry_count += 1
         return processed_entries, failed_entry_count
+
+    def _get_scheduled_rule_job_function(self, account_id, region, rule: Rule) -> Callable:
+        def func(*args, **kwargs):
+            """Create custom scheduled event and send it to all targets specified by associated rule using respective TargetSender"""
+            for target in rule.targets.values():
```

Comment:
Given this function runs in a separate thread. Any exceptions outside the try/catch block get ignored silently. It might be worth catching any unexpected errors and logging them.
(I know it wasn't better before, just remember to track that somewhere so we tackle that in the scheduler rework)